src/main/java/uk/gov/pay/api/service/CapturePaymentService.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.pay.api.service;
+
+import uk.gov.pay.api.auth.Account;
+
+import javax.inject.Inject;
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.core.Response;
+
+public class CapturePaymentService {
+
+    private final Client client;
+    private final ConnectorUriGenerator connectorUriGenerator;
+
+    @Inject
+    public CapturePaymentService(Client client, ConnectorUriGenerator connectorUriGenerator) {
+        this.client = client;
+        this.connectorUriGenerator = connectorUriGenerator;
+    }
+
+    public Response capture(Account account, String chargeId) {
+        Response connectorResponse = captureCharge(account, chargeId);
```

Comment:
This local variable should be removed and the private method inlined, as there is no extra logic done, is just wrapping the http client call. For example could look like this :)
```
public Response capture(Account account, String chargeId) {
        return client
                .target(connectorUriGenerator.captureURI(account, chargeId))
                .request()
                .post(Entity.json("{}"));
    }
```